tests/test_H0.py

```diff
@@ -54,7 +54,7 @@ def test_gross():
     classifier.run_all()
 
     np.testing.assert_equal(analysis.triggers[0][0],
-                            timestamps[-rejected_H0_time])
+                            timestamps[-(rejected_H0_time+integration)])
```

Comment:
It can be helpful for readability of tests to assign these values to variables `exp` for the expected result and `obs` for the observed result

Reply:
I've done this for all tests (except `init`) regardless of whether it was entirely necessary, just to keep a consistency between what is the exp variable and the obs variable. In cases where there are multiple `np.assert` statements, or further clarification needed, I've used `exp_descriptor`.